test/info.py

```diff
@@ -56,7 +56,7 @@
     ("k20dx_mimxrt1020_evk_qspi_if",                True,       0x8000,     "bin"       ),
     ("k20dx_mimxrt1050_evk_hyper_if",               True,       0x8000,     "bin"       ),
     ("k20dx_mimxrt1050_evk_qspi_if",                True,       0x8000,     "bin"       ),
-    ('k20dx_ep_agora_if',                           False,      0x0000,     "bin"       ),
+    ('k20dx_ep_agora_if',                           True,      	0x0000,     "bin"       ),
```

Comment:
Probably will want to fix the whitespace. Looks like a TAB too.

Reply:
@0xc0170 looking at this on my phone it doesn't appear like it did on my laptop

@millerc-ep is this ready for another build?

Reply:
@millerc-ep Spacing still seems a little off to me, might want to review just so it looks clean.